rust-runtime/aws-smithy-checksum-callbacks/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "aws-smithy-checksum-callbacks"
```

Comment:
Maybe just `aws-smithy-checksums`? The callback part seems like an implementation detail.